pull_request.sh

```diff
@@ -48,10 +48,24 @@ else
   annotation_argument=--github-annotate
 fi
 
-"${TRUNK_PATH}" check \
-  --ci \
-  --upstream "${upstream}" \
-  --github-commit "${git_commit}" \
-  --github-label "${INPUT_LABEL}" \
-  "${annotation_argument}" \
-  ${INPUT_ARGUMENTS}
+github_path=git
+# if in test, we need to fake out calling git too
+if [[ ${TRUNK_PATH} == "../local-action/action_tests/stub.js" ]]; then
+  github_path=${TRUNK_PATH}
+fi
```

Comment:
Hmm. I hear what you're saying, and I like the idea, but I don't think we need to do this. Rather, we should be able to, in assert.js, if we're doing the `pull-request-autofix` test case, make assertions about the state of `feature-branch` and `origin/feature-branch`

Reply:
i think the problem is i can't push an empty commit into github - id have to fake a file change and push up a commit which im not super sure how to do

Reply:
but you're not pushing to github - `repo-under-test` is set to push to `src-repo`